File: 1ª Fase 2019/1ª Fase/Competição/B_pramim/solutions/pramim.c
/*
╔══════════════════════════════════════════════════════════════════════════════╗
║ Instituição   :  Faculdade de Tecnologia de São Paulo                        ║
║ Departamento  :  Tecnologia da Informação                                    ║
║ Curso         :  Análise e Desenvolvimento de Sistemas                       ║
║ Autor         :  Lucio Nunes de Lira                                         ║
╠══════════════════════════════════════════════════════════════════════════════╣
║ Competição    :  InterFatecs 2018 - 1ª fase                                  ║
║ Programa      :  Problema B - Um pra você, um pra mim                        ║
║ Linguagem     :  C99                                                         ║
║ Compilador    :  Pelles C (8.00.60)                                          ║
║ Versão        :  A (Rev. 0)                                                  ║
╚══════════════════════════════════════════════════════════════════════════════╝
*/

#include <stdio.h>

int main(void) {
    char p;
    int amigo = 0, n, nota, voce = 0;
    scanf("%c %d", &p, &n);
    for (int i = 0; i < n; i++) {
        scanf("%d", &nota);
        if (p == 'V') {
            voce += nota;
            p = 'A';
        }
        else {
            amigo += nota;
            p = 'V';
        }
    }
    printf("VOCE: %d AMIGO: %d\n", voce, amigo);
    return 0;
}

File: 1ª Fase 2018/1ª Fase/Competição/B_pramim/solutions/pramim.py
'''
╔══════════════════════════════════════════════════════════════════════════════╗
║ Instituição   :  Faculdade de Tecnologia de São Paulo                        ║
║ Departamento  :  Tecnologia da Informação                                    ║
║ Curso         :  Análise e Desenvolvimento de Sistemas                       ║
║ Autor         :  Lucio Nunes de Lira                                         ║
╠══════════════════════════════════════════════════════════════════════════════╣
║ Competição    :  InterFatecs 2018 - 1ª fase                                  ║
║ Programa      :  Problema B - Um pra você, um pra mim                        ║
║ Linguagem     :  Python 3                                                    ║
║ Compilador    :  CPython (3.6.4)                                             ║
║ Versão        :  A (Rev. 0)                                                  ║
╚══════════════════════════════════════════════════════════════════════════════╝
'''

p, n = input().split()
n = int(n)
voce, amigo = 0, 0
for i in range(n):
    nota = int(input())
    if p == 'V':
        voce += nota
        p = 'A'
    else:
        amigo += nota
        p = 'V'
print('VOCE: %d AMIGO: %d' % (voce, amigo))
